docs/api/qiskit-ibm-runtime/dev/qiskit_ibm_runtime.fake_provider.FakeBrooklynV2.mdx

```diff
@@ -44,10 +46,6 @@ python_api_name: qiskit_ibm_runtime.fake_provider.FakeBrooklynV2
 
     This is required to be implemented if the backend supports Pulse scheduling.
 
-    **Return type**
-
-    `Optional`\[`float`]
-
```

Comment:
@Eric-Arellano is this an expected consequence of https://github.com/Qiskit/qiskit-ibm-runtime/pull/2052 ?

Reply:
Yes, note the section `Introduces a regression: type hints for properties` in the PR title. We decided it was worth it